driver/compile.go

```diff
@@ -104,6 +118,16 @@ func compileSingle(ctx context.Context, program ast.Proc, zctx *resolver.Context
 	return newMuxOutput(pctx, leaves, sn), nil
 }
 
+type MultiConfig struct {
+	Custom      compiler.Hook
+	Dir         int
```

Comment:
Can you use zbuf.Order here and if you want an int just call `order.Int()`?
```suggestion
	Order         zbuf.Order
```

Reply:
What happens here is that the Dir come from the inbound /search request, get copied into the MultiConfig, and then is used to create the outbound /worker request. In the inbound request and outbound request, Dir is an integer. What is the advantage of converting the inbound /search Dir into a zbuf.Order and then converting that back into an int for the outbound /worker request? 

This would impact some function signatures, like this one in search.go:
func (s *SearchOp) Run(ctx context.Context, dir int, spc space.Space, output Output) ...

I would like these function signatures to use whatever our most common conventions are.

Reply:
> I would like these function signatures to use whatever our most common conventions are.

Certainly, and this is the point of my comment. I'm trying to establish one way of specifying Order in the code base (see #1505) as we currently have several different ways of doing this. I've seen the translate bool to int function in other packages.

 > What is the advantage of converting the inbound /search Dir into a zbuf.Order and then converting that back into an int for the outbound /worker request?

MultiConfig is not just used w.r.t zqd, its also used in zar zq. This layer should not be concerned with translating api requests off of the wire. Eventually the zqd package will be concerned with this translation. Making this zbuf.Order move us to consistency.

> This would impact some function signatures, like this one in search.go:
func (s *SearchOp) Run(ctx context.Context, dir int, spc space.Space, output Output) ...

Yeah this is an example of an inconsistency around how we specify order that I would like to work towards correcting.

Reply:
OK, I understand the goal now. I changed Dir to Order everywhere but the zapi api calls.